package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --ci --reporters=default --reporters=jest-junit",
```

Comment:
These flags tell circle to generate a junit format xml file with the test results. We still also use the `default` reporter so that human readers can see the test output.